src/components/PayPalButtons.test.tsx

```diff
@@ -332,18 +332,20 @@ describe("<PayPalButtons />", () => {
         const spyConsoleError = jest
             .spyOn(console, "error")
             .mockImplementation();
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        window.paypal!.Buttons = () => {
-            return {
-                close: jest.fn().mockResolvedValue({}),
-                isEligible: jest.fn().mockReturnValue(true),
-                render: jest.fn((element: string | HTMLElement) => {
-                    // simulate adding markup for paypal button
-                    if (typeof element != "string")
-                        element.append(document.createElement("div"));
-                    return Promise.reject("Unknown error");
-                }),
-            };
+        window.paypal = {
+            Buttons() {
+                return {
+                    close: jest.fn().mockResolvedValue({}),
+                    isEligible: jest.fn().mockReturnValue(true),
+                    render: jest.fn((element: string | HTMLElement) => {
+                        // simulate adding markup for paypal button
+                        if (typeof element != "string")
```

Comment:
Please use brackets and use a strict equality check
```suggestion
                        if (typeof element !== "string") {
                            element.append(document.createElement("div"));
                        }
```